src/Native/Unix/System.IO.Compression.Native/pal_zlib.h

```diff
@@ -75,7 +75,7 @@ Initializes the PAL_ZStream so the Deflate function can be invoked on it.
 
 Returns a PAL_ErrorCode indicating success or an error number on failure.
 */
-extern "C" int32_t CompressionNative_DeflateInit2_(
+int32_t CompressionNative_DeflateInit2_(
```

Comment:
Mark these with `DLLEXPORT` while you are on it like it is done in the other files converted to C?

Reply:
Added.